core/src/impl/Kokkos_ViewCtor.hpp

```diff
@@ -322,6 +332,13 @@ KOKKOS_FUNCTION const auto &get_property(
     return view_ctor_prop;
   }
 }
+#if defined(KOKKOS_COMPILER_NVCC) && (KOKKOS_COMPILER_NVCC < 1150)
+// pragma pop is getting a warning from the underlying GCC
+// for unknown pragma if -pedantic is used
+#ifdef __CUDAR_ARCH__
```

Comment:
```suggestion
#ifdef __CUDA_ARCH__
```
Typo in the macro name

Reply:
gah ...